smarts/core/tests/test_smarts_frame_rate.py

```diff
@@ -92,7 +106,7 @@ def smarts():
     agents = {AGENT_1: laner}
     smarts = SMARTS(
         agents,
-        traffic_sims=[SumoTrafficSimulation(headless=True)],
+        traffic_sims=[SumoTrafficSimulation(headless=False)],
```

Comment:
Why run with sumo-gui ?

Reply:
This is what is causing the failing test.